app/Http/Controllers/HomeController.php

```diff
@@ -29,7 +30,8 @@ public function __construct(
         OfficeRepository $office,
         UserRepository $user,
         ReviewBookRepository $review
-    ) {
+    )
+    {
         $this->book = $book;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line